### PR TITLE
CSV: Refresh categories and other minor updates

### PR DESCRIPTION
**Pull request by fbladilo**
Various CSV updates, @fdupont-redhat let me know if you want to include the must-gather as related images and I will also include them on CSV.

### CODE REVIEW

**Comment by fdupont-redhat** [comment]
Can you also add `konveyor` as a keyword?

File: bundle/manifests/konveyor-forklift-operator.v99.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -174,7 +174,7 @@ spec:
     The Konveyor Forklift Operator fully manages the deployment and life cycle of Forklift on OpenShift.
 
     If you experience any issues or have feature requests, please [file an issue in our Github repository](https://github.com/konveyor/forklift-operator/issues)
-  keywords: ['migration']
+  keywords: ["migration", "forklift"]
```
Comment (fdupont-redhat) [inline bundle/manifests/konveyor-forklift-operator.v99.0.0.clusterserviceversion.yaml:177]: Can you also add konveyor as a keyword, please?

Comment (fbladilo) [reply bundle/manifests/konveyor-forklift-operator.v99.0.0.clusterserviceversion.yaml]: done

**Review by fdupont-redhat — APPROVED** [review]
Very well done.

**Pull request merged**